setup_board

```diff
@@ -149,7 +149,6 @@ if [[ "${FLAGS_usepkgonly}" -eq "${FLAGS_TRUE}" ]]; then
   done
 fi
 
-get_board_and_variant $FLAGS_board $FLAGS_variant
```

Comment:
But this should stay, I think. Otherwise BOARD, VARIANT and BOARD_VARIANT will not be initialized.

Reply:
Oh no I removed it entirely instead of moving it back where it was :-/ Will fix.